packages/mobile/src/account/updateAccountRegistration.ts

```diff
@@ -0,0 +1,35 @@
+import networkConfig from 'src/geth/networkConfig'
+
+export interface RegistrationProperties {
+  language?: string | null
+  country?: string
```

Comment:
thanks for the reminder! as we discussed i will double check which solution we want to go for, and keep this implementation for now since it won't be a breaking change if we want to capture the country server side.